### PR TITLE
Update snapshot branch from master

### PR DESCRIPTION
**Pull request by DanHeidinga**

### CODE REVIEW

File: compiler/codegen/CCData.cpp [hunk 1]
```diff
@@ -41,7 +41,6 @@ size_t CCData::dataAlignmentFromBytesAlignment(size_t alignmentBytes)
    return (alignmentBytes + OMR_ALIGNOF(data_t) - 1) / OMR_ALIGNOF(data_t);
    }
 
-inline
```
Comment (DanHeidinga) [inline compiler/codegen/CCData.cpp:44]: @ymanton @0xdaryl I had to remove the `inline` keyword to resolve missing symbol errors:
```
JVMJ9VM011W Unable to load j9jit29: /Users/heidinga/projects/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/amd64/compressedrefs/libj9jit29.so: undefined symbol: _ZN2TR6CCData22byteIndexFromDataIndexEm
```

File: compiler/codegen/OMRCodeGenerator.cpp [hunk 2]
```diff
@@ -285,123 +285,6 @@ OMR::CodeGenerator::initialize()
    cg->createObjectFormat();
    }
 
-
-OMR::CodeGenerator::CodeGenerator() :
```
Comment (DanHeidinga) [inline compiler/codegen/OMRCodeGenerator.cpp:289]: @0xdaryl I removed this snapshot branch specific constructor as I think the `master` changes removed the need for it....

**Comment by DanHeidinga** [comment]
@0xdaryl @TobiAjila thoughts on merging this and the corresponding openj9 snapshot branch PR?

**Review by tajila — APPROVED** [review]

**Comment by tajila** [comment]
@DanHeidinga Daryl mentioned he will also look at these today

File: compiler/codegen/OMRCodeGenerator.cpp (hunk 2 above)
Comment (0xdaryl) [reply compiler/codegen/OMRCodeGenerator.cpp:289]: Yes, that's the correct thing to do.

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**